ui/src/CitationTooltipBody.tsx

```diff
@@ -14,25 +19,39 @@ export class CitationTooltipBody extends React.Component<
   render() {
     return (
       <div className="tooltip-body citation-tooltip-body">
-        <div className="tooltip-body__section tooltip-body__label citation-tooltip-body__header">
-          This citation was matched to {this.props.paperIds.length}
-          {this.props.paperIds.length > 1 ? " papers" : " paper"}:
-        </div>
+        <Box padding="8px 8px 0">
```

Comment:
I think so.

* `Box` is a `@material-ui` specific component, a sort of catch-all for setting padding, margin and other layout-esque css that's component specific.  Given we're using `@material-ui` I'd suggest opting into their APIs. That said if we think we're likely to abandon `@material-ui` in the future, then we should likely err away from using it.  
*  I love using `display: grid` over `display: flex` lately, as it feels like it's more precise.  Here I'm using `@material-ui`'s `Grid`, which ironically uses `display: flex`.  Again I think it's just a matter of choosing whether we want to "opt-in" to `@material-ui`'s general layout mechanism or not.

The components provided by `@material-ui/` seem quite powerful, and saved me a fair amount of time in composing this PR.  I think it'd be a fine choice for a component library for the project, and seems theme-able to the extent that we can make sure it doesn't end up looking like a Google product.

Let me know what you think!

Reply:
Continuing my thought here a bit.

It seems we're leaning away from `@material-ui` in some respects.  The conversation around using `less` over `makeStyles`, for instance, shows that we've opted out of the canonical styling mechanism provided by `@material-ui`.   

I'd personally advise against this -- as I think there's some gotchas to going this route:

- Having two ways to style things creates a bit of confusion for implementers (as illustrated here). Should one use `less` or `makeStyles`.  Is the answer always `less`? It's easy to opt into `makeStyles` given it's well documented in `@material-ui`s docs.  Are we sure `less` is providing enough value here that it's worth using here? If it is, how do we clearly document when to use one or the other?

- Using both will mean we'll need to do some acrobatics to keep certain things in sync, particularly if / when we create a theme for `@material-ui`'s components.  Hopefully that theme can be something we source in both the `less` files and the `@material-ui` theme, otherwise we'll have two sources of truth for our "layout parameters" (padding values, colors, font-sizes, etc).  

That said, this is all stuff to think about post this week's deadline.  I'll refactor to remain consistent with where we're at now, and maybe you can carve out some time to assess this in early 2020!

Reply:
Updated to use `less`.  Let's chat about this in the new year! 🎉